apache-activemq-mixin/dashboards/apache-activemq-logs-overview.libsonnet

```diff
@@ -0,0 +1,35 @@
+local g = import 'github.com/grafana/grafonnet/gen/grafonnet-latest/main.libsonnet';
+local logsDashboard = import 'github.com/grafana/jsonnet-libs/logs-lib/logs/main.libsonnet';
+
+{
+  grafanaDashboards+::
+    {
+      local activemqLogs =
+        logsDashboard.new(
+          'Apache ActiveMQ logs',
+          datasourceName='loki_datasource',
+          datasourceRegex='',
+          filterSelector=$._config.filterSelector,
+          labels=['job', 'activemq_cluster', 'instance', 'level'],
+          formatParser=null,
+          showLogsVolume=true
+        )
+        {
+          panels+:
+            {
+              logs+:
+                // ActiveMQ logs already have timestamp
+                g.panel.logs.options.withShowTime(false),
+            },
+          dashboards+:
+            {
+              logs+: g.dashboard.withLinksMixin($.grafanaDashboards['apache-activemq-cluster-overview.json'].links)
```

Comment:
Does this actually add links to all other dashboards? This whole file format looks different than what I'm used to seeing for dashboards libsonnets.

Reply:
Yes it does! It's a bit different as the logs dashboard is generated and I have to add those bits in to make sure the dashboard comes with the correct information when generating it (eg: copying the links from another dashboard). I followed the example for kubernetes/clickhouse logs dashboards to make this.

Reply:
good to resolve